### DISCUSSION TITLE
Modal Form too small with unnecessary validation

### DISCUSSION THREAD

**Discussion by PaulWaldo** [fyne-io/fyne#3811] [Q&A] [answered]
I want to create a simple form for the user to enter some data.  A modal is preferred, so I use this
```
form := dialog.NewForm("Title", "Confirm", "Dismiss",[]*widget.FormItem{
	{Text: "Server", Widget: serverUrlEntry, HintText: "URL of your server"},
},func(b bool) {},w)
```
This gives me a tiny window with quite a bit of space used for the validation markers that I don't use
![CleanShot 2023-04-11 at 18 30 56@2x](https://user-images.githubusercontent.com/12105941/231303016-7b47d707-c07d-4cea-a4a0-a77f1d3b3f19.png)

I like the clean look of the form in `fyne_demo`
![CleanShot 2023-04-11 at 18 31 39@2x](https://user-images.githubusercontent.com/12105941/231303121-4e746437-40e0-451a-a8b9-2d88aa14026a.png)

but my reading of the docs and code indicate it would be use for an entire canvas.  What would be the best way to combine the modality of the former with the size and simplicity of the latter?

**Comment by andydotxyz** [comment, marked as answer]
To make a dialog bigger than the MinSize it calculates use the `Resize` method of the dialog returned from `dialog.NewXxx`.

Validations are automatically added for data binding, you can turn them off by setting the `Validator` field to nil after binding.

**Reply by PaulWaldo** [reply]
Thank you @andydotxyz, that did the trick!
